netbout-web/src/main/java/com/netbout/email/EmAlias.java

```diff
@@ -112,7 +143,88 @@ public void email(final String email) throws IOException {
     }
 
     @Override
+    public void email(final String email, final String urn, final Bout bout)
+        throws IOException {
+        this.origin.email(email);
+        this.send(email, urn, bout);
+    }
+
+    @Override
     public Inbox inbox() throws IOException {
         return new EmInbox(this.origin.inbox(), this.postman, this.name());
     }
+
```

Comment:
@mstzn36 Let's put this entire block in another class and get rit of the PMD and checkstyle warnings